### ISSUE TITLE
We should be able to tweets via Screamout.

### ISSUE THREAD

**Issue by gautamrege** [joshsoftware/intranet#81]
When we add stuff to screamout, it's not only very helpful for the team but can also be used to tweet from the company handle, mentioning the twitter handle of the person who added it. 

For example, if Deepak screamed out "Git reset explained", the admin could tweet from the intranet page using josh software twitter handle "Git reset explained <link> /cc @spidergears".

**Comment by spidergears** [comment]
Or we can setup a tweet bot... provide we trust that the content being screamed out is worth it.

**Comment by gautamrege** [comment]
Yes - absolutely. For starters, I want it moderated. Later on, we can automate it using tweet bot :)
